src/styles/_maps.scss

```diff
@@ -12,7 +12,8 @@ $theme-colors: map-merge(
     'gray-100': $gray-100,
     'gray-modal': $gray-modal-bg,
     'black': $black,
-    'white': $white
+    'white': $white,
+    'table-header': $gray-modal-border
```

Comment:
Are you sure it should be global theme color?

Reply:
@asvishnyakov by adding a color to a $theme-colors variable we are making them available to use in utility classes for border/text/background colors